osu.Game.Tests/Online/TestSceneOnlinePlayBeatmapAvailabilityTracker.cs

```diff
@@ -143,7 +143,11 @@ private static BeatmapInfo getTestBeatmapInfo(string archiveFile)
                 var beatmap = decoder.Decode(reader);
 
                 info = beatmap.BeatmapInfo;
-                info.BeatmapSet.Beatmaps = new List<BeatmapInfo> { info };
+                info.Metadata = info.Metadata;
```

Comment:
Is this intentional? It's also done in `TestBeatmap` as well.

Reply:
Unintentional, fixed.